### DISCUSSION TITLE
Vim yank registers not working?

### DISCUSSION THREAD

**Discussion by mwinters0** [zed-industries/zed#16891] [Feature Requests]
I have vim mode enabled and have tried every variation of `use_system_clipboard`, but I can't get yank registers to work.  It looks like this feature exists: https://github.com/zed-industries/zed/pull/12895 .  But I'm a new user and can't tell whether Zed is broken or I am.

When I do `Vjj"ay` and then `"ap` or `"aP` nothing is pasted.  Same for any other register.  The clipboard settings only seem to affect whether / how yanks make it to the system clipboard.  Default registers (`yy` and `p`) do work.

This is on v0.149.5-1 under KDE on Arch.  I've tried this with a fresh install (switching from Flatpak build to Pacman) and with my settings entirely default except for enabling vim_mode, but got the same behavior.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by mwinters0** [reply]
Sorry, I'll not be joining a chat platform that requires age identification.  Please consider using Zulip to better support your community.
